jsm.go

```diff
@@ -987,5 +1112,22 @@ func getJSContextOpts(defs *jsOpts, opts ...JSOpt) (*jsOpts, context.CancelFunc,
 		o.pre = defs.pre
 	}
 
+	// 1 normal try plus the number of retries.
+	o.maxTries++
+	if o.retryBackoff == time.Duration(0) {
+		o.retryBackoff = 10 * time.Millisecond
```

Comment:
10ms maybe too short for a delay, how about 100ms?

```suggestion
		o.retryBackoff = 100 * time.Millisecond
```